packages/e2e-tests/specs/editor/various/navigable-toolbar.test.js

```diff
@@ -1,104 +0,0 @@
-/**
- * WordPress dependencies
- */
-import { createNewPost, pressKeyWithModifier } from '@wordpress/e2e-test-utils';
-
-async function isInBlockToolbar() {
-	return await page.evaluate( () => {
-		return !! document.activeElement.closest(
-			'.block-editor-block-toolbar'
-		);
-	} );
-}
-
-describe( 'Block Toolbar', () => {
-	beforeEach( async () => {
-		await createNewPost();
-	} );
-
-	describe( 'Contextual Toolbar', () => {
-		it( 'should not scroll page', async () => {
-			while (
-				await page.evaluate( () => {
-					const { activeElement } =
-						document.activeElement?.contentDocument ?? document;
```

Comment:
I think this part needed to get moved over to the playwright test here and in `scrollTopBefore`